test/app-luatest/override_misc_test.lua

```diff
@@ -52,3 +52,21 @@ g.test_initializing = function(g)
     local exp = expected_output('socket')
     t.assert_equals(res, exp)
 end
+
+g.test_no_package_loaded = function(g)
+    local load_cnt = 0
+    local foo = {whoami = 'foo'}
+    package.preload.foo = function()
+        local loaders = require('internal.loaders')
+        load_cnt = load_cnt + 1
+        if load_cnt < 3 then
```

Comment:
I suggest to use `LOAD_LIMIT` constant here and in `assert_equals()` below.

Reply:
Okay, done. Diff for this and several close comments:

```diff
diff --git a/test/app-luatest/override_misc_test.lua b/test/app-luatest/override_misc_test.lua
index f4814a2d8..96fa750c1 100644
--- a/test/app-luatest/override_misc_test.lua
+++ b/test/app-luatest/override_misc_test.lua
@@ -66,20 +66,25 @@ g.test_initializing = function(g)
     t.assert_equals(res, exp)
 end
 
-g.test_no_package_loaded = function(g)
+g.test_no_package_loaded = function()
+    local LOAD_LIMIT = 3
+    local LOAD_ATTEMPTS = LOAD_LIMIT + 1
+
     local load_cnt = 0
     local foo = {whoami = 'foo'}
     package.preload.foo = function()
         local loaders = require('internal.loaders')
         load_cnt = load_cnt + 1
-        if load_cnt < 3 then
+        if load_cnt < LOAD_LIMIT then
             loaders.no_package_loaded.foo = true
         end
         return foo
     end
-    for i = 1, 10 do
+
+    for _ = 1, LOAD_ATTEMPTS do
         require('foo')
     end
+
     package.preload.foo = nil
-    t.assert_equals(load_cnt, 3)
+    t.assert_equals(load_cnt, LOAD_LIMIT)
 end
```

Reply:
Nice, thanks!